src/components/header/index.js

```diff
@@ -12,14 +12,15 @@ const Logo = styled.img`
   max-height: 100px;
 `;
 
-const Header = styled.div`
+const Header = styled.a`
   text-align: center;
   margin-bottom: 16px;
+  display: block;
 `;
 
 export default () =>
   process.env.REACT_APP_LOGO || process.env.REACT_APP_NAME ? (
-    <Header>
+    <Header href={process.env.REACT_APP_LINK ? process.env.REACT_APP_LINK : undefined}>
```

Comment:
Shouldn't need the ternary, since if `process.env.REACT_APP_LINK` isn't set, it's `undefined`, which is what you're setting `href` to here anyways. If you're worried about not having a fallback, you could use `#` instead:

```jsx
<Header href={process.env.REACT_APP_LINK || "#"}>
```